app/index.js

```diff
@@ -77,6 +78,11 @@ function onListening() {
   const addr = server.address();
   const bind = typeof addr === 'string' ? `pipe ${addr}` : `port ${addr.port}`;
   log.info(`razeedash-api listening on ${bind}`);
+
+  // if AUTH_MODEL is not defined, turn off query API.
```

Comment:
comment logic seems backward from the implementation. Looks like you enable the API if the AUTH_MODEL is defined, rather  than disabling it if undefined

Reply:
revised the comment.